lib/Braintree/Configuration.php

```diff
@@ -289,6 +289,8 @@ public function assertHasAccessTokenOrKeys()
                 throw new Exception\Configuration('Braintree\\Configuration::publicKey needs to be set.');
             } else if (empty($this->_privateKey)) {
                 throw new Exception\Configuration('Braintree\\Configuration::privateKey needs to be set.');
+            } else if(empty($this->_merchantId) || empty($this->_environment) || empty($this->_publicKey) || empty($this->_privateKey)){
```

Comment:
This will never get here. It'll run threw one of the other if conditions fist and throw an exception there.

Reply:
@Gicehajunior Could you explain the problem you're trying to solve here? I agree with @crookedneighbor that the code you've added will never be executed since if this condition is true, one of the previous conditions must also be true.